.circleci/config.yml

```diff
@@ -435,77 +435,62 @@ jobs:
           command: build noir-projects
           aztec_manifest_key: noir-projects
 
-  boxes-files:
-    machine:
-      image: default
-    resource_class: medium
+  yarn-project-pre-join:
+    docker:
+      - image: cimg/base:2023.09
+    resource_class: small
     steps:
-      - *checkout
-      - *setup_env
       - run:
-          name: "Build"
-          command: build boxes-files
-          aztec_manifest_key: boxes-files
+          name: "Noop"
+          command: echo Noop
 
-  yarn-project-base:
-    machine:
-      image: default
-    resource_class: large
+  yarn-project-x86_64:
+    docker:
+      - image: aztecprotocol/alpine-build-image
+    resource_class: small
     steps:
       - *checkout
       - *setup_env
       - run:
           name: "Build"
-          command: build yarn-project-base | add_timestamps
-          aztec_manifest_key: yarn-project-base
+          command: cond_spot_run_build yarn-project 64
+          aztec_manifest_key: yarn-project
 
-  yarn-project:
-    machine:
-      image: default
-    resource_class: large
+  yarn-project-arm64:
+    docker:
```

Comment:
I'm slightly in favour of just having project: and project-arm64:, but not a big deal